release-notes/release-notes509.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: TinyMCE 5.0.9
+title_nav: TinyMCE 5.0.9
+keywords: releasenotes newfeatures deleted technologypreview bugfixes knownissues
+---
+
+## Overview
+
+The Release Notes provide high-level coverage of the improvements and additions that have been implemented in TinyMCE 5.0.9 and document known problems in this release, as well as important bug fixes, deprecated functionality, and other details.
+
+TinyMCE 5.0.9 release adds improvements to the TinyMCE editor to improve the overall user experience.
+
+> **Note:** TinyMCE 5.0.8 community was not released to enterprise due to blockers found during QA. As such TinyMCE 5.0.9 enterprise contains changes from both TinyMCE 5.0.8 and 5.0.9 community releases.
```

Comment:
`TinyMCE 5.0.8 community was not released to enterprise due to blockers found during QA` sounds... not quite formal enough to me? Or rather, I'm not sure if `blockers` is general terminology. Maybe `...due to issues found during QA.` instead?